lib/sqewer.rb

```diff
@@ -24,6 +26,18 @@ def self.submit!(*jobs, **options)
     Sqewer::Submitter.default.submit!(*jobs, **options)
   end
   
+  def self.default_connection_pool
```

Comment:
Can we create an accessor pair (`client_pool` / `client_pool=`) instead, to adhere by the principle of least surprise? We can then also return a NullPool from the reader if nothing was configured